src/wolfgame.py

```diff
@@ -2074,6 +2067,7 @@ def fday(cli, nick, chan, rest):
 
 # Specify force = "nick" to force nick to be lynched
 @handle_error
```

Comment:
No it doesn't. it wraps it for things called using the stub, but the original method is not wrapped in impl.